src/main/resources/swagger.api/mod-search.yaml

```diff
@@ -217,31 +225,20 @@ components:
       schema:
         type: string
         pattern: '[a-zA-Z]{3}'
-    search-cql-query:
-      name: query
+    cql-search-request:
+      name: cqlSearchRequest
       in: query
       description: A CQL query string with search conditions.
-      schema:
-        type: string
       required: true
-    search-pageable-offset:
-      name: offset
-      in: query
-      description: Skip over a number of elements by specifying an offset value for the query
       schema:
-        default: 0
-        minimum: 0
-        maximum: 2147483647
-        type: integer
-    search-pageable-limit:
-      name: limit
+        $ref: '#/components/schemas/cqlSearchRequest'
+    cql-facet-request:
+      name: cqlFacetRequest
       in: query
-      description: Limit the number of elements returned in the response
+      description: A CQL query string with search conditions.
```

Comment:
Update description to mention `a list of facet names`

Reply:
Renamed to `A CQL query string with search conditions and facet names.`